Userland/Libraries/LibVideo/VP9/Parser.h

```diff
@@ -12,6 +12,7 @@
 #include <AK/Span.h>
 #include <AK/Vector.h>
 #include <LibGfx/Forward.h>
+#include <LibVideo/Color/CodingIndependentCodePoints.h>
```

Comment:
unused include as it seems

Reply:
The enum was previously defined in VP9/Enums.h, this changes it over to be the reusable enum from CodingIndependentCodePoints.h since at least AV1 can also use this enum in its parser later.